.github/workflows/build-docs.yml

```diff
@@ -0,0 +1,20 @@
+name: Docs
```

Comment:
I don't know if this will actually work because of the C code but we will find out I guess? Did you run `make serve-docs` locally?